BREAKING.md

```diff
@@ -27,6 +28,9 @@ the npm scopes.
 ### IContainerRuntime property removed
 - the `existing` property from `IContainerRuntime` has been removed.
 
+### Property removed from the Container class
+- the `existing` property from `Container` has been removed.
+
```

Comment:
I think it deserves to mention that legacy way of creating files is deprecated in this release, and maybe talk about way to enable it for now through environment property on config?